snapshots/biodiversity/2023-01-11/cherry_blossom.py

```diff
@@ -0,0 +1,30 @@
+"""This script has been autogenerated by `walkthrough snapshot`."""
```

Comment:
Could you explain here in the docstring the instructions to obtain the data file?